plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -104,7 +104,9 @@ export default {
             }
 
             get title() {
-              return this.channel.escapedTitle;
+              return this.channel.description
+                ? htmlSafe(this.channel.description)
```

Comment:
I'm not quite sure here but is it safe to mark the channel's description as `htmlSafe`? 

cc @jjaffeux

Reply:
yes you should use `escapedDescription` @romanrizzi added